pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/internal/pex/ThinPexGenerator.java

```diff
@@ -45,26 +44,26 @@
     private final Map<String, String> extraProperties;
 
     public ThinPexGenerator(
-            Project project,
-            List<String> pexOptions,
-            EntryPointTemplateProvider templateProvider,
-            Map<String, String> extraProperties) {
+        Project project,
+        List<String> pexOptions,
+        EntryPointTemplateProvider templateProvider,
+        Map<String, String> extraProperties) {
```

Comment:
This line (and other arguments above it) was indented more in the original (IDE default I believe) to be distinguishable from the first line of the method code that follows next. So, I'm not sure about the usefulness of this dedent.

Again, something where we need to figure out what's the IDE default.

Reply:
I have a clean install of IntelliJ. We can apply a strict style guide after this PR and make it automated.